code/game/mecha/mech_fabricator.dm

```diff
@@ -1,46 +1,59 @@
+#define EXOFAB_BASE_CAPACITY 200000
+#define EXOFAB_CAPACITY_PER_RATING 50000
+#define EXOFAB_EFFICIENCY_PER_RATING 0.15
+#define EXOFAB_SPEED_UPGRADE_MULTIPLIER 0.2
+
+/**
+  * # Exosuit Fabricator
+  *
+  * A machine that allows for the production of exosuits and robotic parts.
+  */
 /obj/machinery/mecha_part_fabricator
-	icon = 'icons/obj/robotics.dmi'
-	icon_state = "fab-idle"
 	name = "exosuit fabricator"
 	desc = "Nothing is being built."
+	icon = 'icons/obj/robotics.dmi'
+	icon_state = "fab-idle"
 	density = TRUE
 	anchored = TRUE
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 20
 	active_power_usage = 5000
+	// Settings
+	/// Bitflags of design types that can be produced.
+	var/allowed_design_types = MECHFAB
+	/// List of categories to display in the UI. Designs intended for each respective category need to have the name in [/datum/design/category]. Defined in [Initialize()][/atom/proc/Initialize].
+	var/list/categories = null
+	/// Unused. Ensures backwards compatibility with some maps.
+	var/id = null
```

Comment:
Why not just remove the `id` override in the other maps? Its only Meta which has the issues, so it shouldnt be a major diff to remove one property.

Reply:
I was under the impression the mapping freeze meant no touching the stations at all?

Reply:
Only BoxStation